src/libcore/intrinsics.rs

```diff
@@ -1393,3 +1406,25 @@ extern "rust-intrinsic" {
     #[cfg(not(stage0))]
     pub fn nontemporal_store<T>(ptr: *mut T, val: T);
 }
+
+// Simple bootstrap implementations for stage0 compilation
+/// Returns the minimum of two `f32` values.
+#[cfg(stage0)]
+pub unsafe fn minnumf32(x: f32, y: f32) -> f32 {
+    (if x < y || y != y { x } else { y }) * 1.0
```

Comment:
Shouldn't you copy the deleted comments from `libcore` at least once here?

Reply:
Good call. Even though these are only used for bootstrapping, they still ought to have that correctness warning.

Reply:
Okay, I was wondering whether they were necessary if just used for bootstrapping. I'll add them back!